### PR TITLE
[11.0] [ADD] account_reconciliation_widget_partial

### PR DESCRIPTION
**Pull request by etobella**
This addons allow to change the amount of a reconciliation line in order to
partially reconcile it.

Usage
=====

On reconciliations, an edit button is created.
Once pressed, you can modify the amount.
By default, it will propose the difference between the original amount and the
current balance if allowed.

@jarroyomorales @jbeficent 

reconciliation_partial

### CODE REVIEW

File: account_reconciliation_widget_partial/static/src/less/account_reconciliation.less [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        .strike_amount {
+            text-decoration: line-through;
+        }
+    }
+    .o_reconciliation_line .match{
+        .line_info_edit {
+            display: none!important;
+        }
+    }
+}
```
Comment (MiquelRForgeFlow) [inline account_reconciliation_widget_partial/static/src/less/account_reconciliation.less]: blank line

File: account_reconciliation_widget_partial/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Reconcile Partial',
```
Comment (MiquelRForgeFlow) [inline account_reconciliation_widget_partial/__manifest__.py]: update name

**Comment by etobella** [comment]
@mreficent Comments attended

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

You know, in v13 this module won't be needed 😜

**Comment by etobella** [comment]
in V13 we will make the greatest reconciliation widget :stuck_out_tongue_closed_eyes:

**Review by AdriaGForgeFlow — APPROVED** [review]
Functional test and code review :+1:

**Pull request merged**